core-it-suite/pom.xml

```diff
@@ -166,9 +162,10 @@ under the License.
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <includes>
-            <include>**/IntegrationTestSuite.java</include>
+            <include>**/MavenIT*.java</include>
           </includes>
-          <forkMode>never</forkMode>
+          <forkCount>1</forkCount>
+          <reuseForks>true</reuseForks>
```

Comment:
Question: We have two profiles: `embedded` and none. Does this change the behavior? Since non-embedded requires much more time.